### DISCUSSION TITLE
Problem Nginx Proxy Manager and /account

### DISCUSSION THREAD

**Discussion by DarthP** [hrfee/jfa-go#336] [Q&A]
Hello everyone,

I have installed jfa-go on my Synology. This also runs in a Docker container

Port: is : 8056. 

I can also access the admin page, but https://register.tv.forexample.tw/my/account does not work.

everything concerning /my or /invite does not work.

I use the Nginx Proxy Manager as proxy manager. The IP to the jfa-go is: http://192.168.10.75:8056

Screenshots below:

![Photo1](https://github.com/hrfee/jfa-go/assets/89332484/ffbbf191-899f-408a-b900-5f7dbf3dddd2)
![Photo2](https://github.com/hrfee/jfa-go/assets/89332484/61dced07-507e-4f0e-b2a4-a0b332c5f37e)
![photo3](https://github.com/hrfee/jfa-go/assets/89332484/f947f5ff-99dd-44a5-ba5d-d3fb79a05619)

**Comment by jaiovi** [comment]
Same issue as you. Have you fixed it yet?

**Comment by hrfee** [comment]
I'm not sure about this one, the same type of config (minus SSL, but that shouldn't have an effect) works for me on 0.5.1 and the most recent commit.
![Screenshot 2024-07-22 at 15-41-58 Nginx Proxy Manager](https://github.com/user-attachments/assets/637f25d5-153c-4f03-ba2e-b7d842cbb950)
